graftm/run.py

```diff
@@ -241,7 +241,7 @@ def graft(self):
        
         # If merge reads is specified, check that there are reverse reads to merge with
         if self.args.merge_reads and not hasattr(self.args, 'reverse'):
-            logging.error("--merge requires --reverse to be specified")
+            logging.error("--merge_reads requires --reverse to be specified")
```

Comment:
does this check need to happen? I'd just raise a Excception("Programming error") at this point, don't worry about a nice exit message because the user should not ever get here any more.